cypress/e2e/navigation.cy.ts

```diff
@@ -0,0 +1,41 @@
+describe('Navigation tests', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should navigate to search page, type a query, navigate to chain details, and go back to home on breadcrumb', () => {
+    cy.getByTestId('search-bar').click();
+    cy.url().should('include', '/search');
+
+    cy.getByTestId('search-bar').find('input').click().type('Invalid chain name');
+    cy.get('input').should('have.value', 'Invalid chain name');
+    cy.getByTestId('data-not-found').should('be.visible');
+
+    cy.getByTestId('search-bar').find('input').clear();
+    cy.get('input').should('have.value', '');
+
+    cy.getByTestId('search-bar').find('input').click().type('324');
```

Comment:
You can type directly in the input, there's no need to click it.
```suggestion
    cy.getByTestId('search-bar').find('input').type('324');
```